modules/tinymce/src/themes/silver/main/ts/ui/core/color/Options.ts

```diff
@@ -86,16 +110,25 @@ const register = (editor: Editor): void => {
 
 const getColorCols = option('color_cols');
 const hasCustomColors = option('custom_colors');
-const getColors = option<Menu.ChoiceMenuItemSpec[]>('color_map');
+const getColors = (editor: Editor, id: string): Menu.ChoiceMenuItemSpec[] => {
```

Comment:
This fallback behavior is not fully covered in tests. Setup an editor with foreground only see if background falls back. Setup with background only see if foreground falls back etc. You could mutate the options directly I guess. Though in theory these are immutable options.

Reply:
Yeah, I'm not keen on the idea of mutating these. Doubly so since some values are calculated beforehand, using the values that exist then, such as the cols ( even though those wouldn't change anything here )